wallet/test/Test/Pos/Wallet/MigrationSpec.hs

```diff
@@ -285,20 +290,34 @@ instance HasConfiguration => Arbitrary WalletStorage_v3 where
 
 
 spec :: Spec
-spec = withCompileInfo def $ withDefConfigurations $ \_ ->
-    describe "Migration to latest version can be reversed" $ do
-      it "(WalletStorage_v2) migrating back results in the original" $ property prop_backMigrate_v2
-      it "(WalletStorage_v3) migrating back results in the original" $ property prop_backMigrate_v3
+spec = do
+    resizeTests $ do
+        describe "Migration to latest version can be reversed" $ do
+            prop
+                "(WalletStorage_v2) migrating back results in the original"
+                prop_backMigrate_v2
+            prop
+                "(WalletStorage_v3) migrating back results in the original"
+                prop_backMigrate_v3
+    describe "Can load the 1.1.0 database" $ do
+        it "can load" $ do
+            db <- openState False "test/wallet-db-1.1.1/"
+            ws <- runReaderT askWalletSnapshot db
+            force ws `shouldNotBe` def
```

Comment:
We `force` it to ensure there aren't any `_|_`s hanging around. And then we say it shouldn't be `def` because the thing will give you back a `Default` of the `WalletStorage` if the file path doesn't exist (d'oh)

Reply:
This test case could probably use a comment to that effect. I know that I, for one, plan on immediately forgetting the details of this stuff :)